templates/js-cypress/main.js

```diff
@@ -0,0 +1,55 @@
+import { Actor } from 'apify';
+import fs from 'fs';
+import cypress from 'cypress';
+import { globby } from 'globby';
+
+await Actor.init();
+
+// todo: be able to send everything in input
+const { video, defaultCommandTimeout, viewportHeight, viewportWidth, ...rest } = await Actor.getInput();
```

Comment:
Since we don't use the parameters elsewhere, we can pass the full input as config.